### ISSUE TITLE
Nodejs support

### ISSUE THREAD

**Issue by crofty** [SevereOverfl0w/high#34]
Hi.
Is Clip expected to work with nodejs?  I'm trying to include clip in a project, compiled with shadow-cljs, targeting a node-script, and I'm getting the following error when I add the clip require:

```
SHADOW import error .shadow-cljs/builds/inform/dev/out/cljs-runtime/juxt.clip.impl.core.js

script.js:64
  /* ignore this, look at stacktrace */ fn.call(global, require, module, __filename, __dirname);
                                           ^
TypeError: Cannot read properties of undefined (reading 'prototype')
```

It looks like it's failing at [this part](https://github.com/juxt/clip/blob/master/src/juxt/clip/impl/core.cljc#L153) of the code.

Is it expected to work on nodejs or is that unsupported?

**Comment by SevereOverfl0w** [comment]
It should be supported, but isn't regularly tested.  https://github.com/juxt/clip/blob/master/src/juxt/clip/impl/core.cljc#L154 is definitely not right, that's going to be java only.

Should it be `js/Object` in JavaScript?

**Comment by dula2000** [comment]
Clip is a library for working with the clipboard on the client-side, specifically in web browsers. It is not designed to work with Node.js, as Node.js does not have access to the clipboard. Instead, you should look for a Node.js library that provides similar functionality for working with the clipboard on the server-side.

**Comment by SevereOverfl0w** [comment]
Should be fixed in 0.28.0, enjoy!

**Comment by crofty** [comment]
Great. Thank you!
